tests/test_timeseriesflattener/test_feature_spec_objects.py

```diff
@@ -144,6 +144,18 @@ def test_resolve_multiple_fn_to_str():
     assert "maximum" in pred_spec_batch[0].get_col_str()
 
 
+def test_lookbehind_days_handles_floats():
+    """Test that resolve_multiple_fn is converted to str correctly."""
```

Comment:
Docstring seems out of date? 😉

Reply:
Oops. ;) Will you fix this as you're already committing minor changes?

Reply:
Sure thing 👍